#Force Control Yaml
sin_freq_rps: 6.3
critical_freq_rps: 32.
# sin_amplitude: 0.
# chirp_offset: -8.
sin_amplitude: 1.5
sin_offset: -0.
sin_duration: 230.
sin_mode: 6
# exo_mass: 0.0
exo_mass: 1.35
# load_mass: 0.5
load_mass: 0.0
# load_mass: 2.2
# load_mass: 4.4

distance: 0.45

ignore_cuff: 1
amp_factor: 2.
lag_zero_cuff: 6000.

lag_pole_cuff: 200.

ignore_damp: 1
vir_damp: 1.
lag_zero_damp: 6000.
lag_pole_damp: 200.

ignore_rand: 1
lag_zero_rand: 6000.
lag_pole_rand: 32.

serial_port: 0